Readme.md

```diff
@@ -236,51 +237,91 @@ You can update your installation's permissions by following these steps:
 Note: We're working on making this process easier in the future with a prompt from the Teams integration itself – stay tuned!
 
 ### Personal app experience
-As part of personal app experience, you can now subscribe to your repositories and receive notifications for:
-- issues
-- pull requests
-- discussions
-- commits
+With our GitHub app, you also get a personal app in MS Teams where you can subscribe to your repos and track your work. The notifications will be posted in the GitHub personal chat that is visible only for you. All the commands and notifications available in your channel are also available in Personal chat for GitHub.
 
-All the commands available in your channel are now available for Personal chat for GitHub.
-<p align="left"><img width="500" alt="PR Card" src="images/PersonalApp.PNG"></p>
+<p align="left"><img width="500" alt="Personal App" src="images/PersonalApp.PNG"></p>
 
-You can also signin to GitHub with Personal app chat and avoid separate signin in your channels.
+In the personal app, threading is not supported by MS Teams. You get the same notifications in personal chat but they are not grouped under parent cards. Instead, you get independent notifications.
+<p align="left"><img width="500" alt="Personal Notifications" src="images/PersonalNotifications.png"></p>
 
 ### Schedule Reminders
-You can now schedule reminders for pending pull requests. With this feature you can now get periodic reminders of pending pull requests as part of your channel or personal chat.
+You can configure schedule reminders for pending pull requests. With this feature you get periodic reminders of pending pull requests as part of your channel or personal chat.
 <p align="left"><img width="500" alt="Schedule reminders" src="images/ScheduleReminders.PNG"></p>
 
+#### What's new with Schedule reminders?
+We made significant improvements to schedule reminders experience. 
+- The old way of configuring reminders via commands is now gone.
+- We have introduced a first class UI to configure your pending pull request reminders. And as part of this UI, we added a bunch of advanced controls that will help you in fine tuning your reminders. 
+- We relaxed the administrator requirement to configure reminders. Now, any contributor can schedule reminders in MS Teams for the repositories he/she has access.
```

Comment:
nit: Do we need to change 'in MS Teams for the repositories he/she has access.' to 'in MS Teams for the organisation he/she has access.' ?